packages/connectivity/src/scp-cf/resilience/resilience-options.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  CircuitBreakerOptions,
+  defaultCircuitBreakerOptions
+} from './circuit-breaker-options';
+
+/**
+ * Options to configure resilience when fetching destinations.
+ * @deprecated
```

Comment:
[req] Mention the mitigation => use resilience property on XYZ instead.

Reply:
Thanks for the reminder. I added the links to the new options.